packages/venia-ui/lib/components/Logo/logo.js

```diff
@@ -18,14 +20,23 @@ const Logo = props => {
     const { height, width } = props;
     const classes = mergeClasses({}, props.classes);
 
+    const talonProps = useLogo({
+        query: GET_LOGO_DATA,
+        veniaLogo: { logo },
```

Comment:
This is more of a default, let's call it `defaultSrc` and also we don't need to wrap it in an object. 

```suggestion
        defaultSrc: logo,
```

Reply:
Added commit with your suggestion.